tests/MongoDB.Bson.Tests/Specifications/bson-corpus/BsonCorpusTestRunner.cs

```diff
@@ -0,0 +1,358 @@
+            {
+                dEJ = UnescapeUnicodeCharacters(degenerateExtendedJson.AsString.Replace(" ", ""));
+            }
+
+            string rEJ = null;
+            if (test.TryGetValue("relaxed_extjson", out var relaxedExtendedJson))
+            {
+                rEJ = UnescapeUnicodeCharacters(relaxedExtendedJson.AsString.Replace(" ", ""));
+            }
+
+            if (cB != null)
+            {
+                EncodeBson(DecodeBson(cB)).Should().Equal(cB, "native_to_bson( bson_to_native(cB) ) = cB");
+                EncodeCanonicalExtendedJson(DecodeBson(cB)).Should().Be(cEJ, "native_to_canonical_extended_json( bson_to_native(cB) ) = cEJ");
+                if (rEJ != null)
+                {
+                    EncodeRelaxedExtendedJson(DecodeBson(cB)).Should().Be(rEJ, "native_to_relaxed_extended_json( bson_to_native(cB) ) = rEJ");
+                }
+            }
+
+            if (cEJ != null)
+            {
+                EncodeCanonicalExtendedJson(DecodeExtendedJson(cEJ)).Should().Be(cEJ, "native_to_canonical_extended_json( json_to_native(cEJ) ) = cEJ");
+                if (!test.GetValue("lossy", false).ToBoolean())
+                {
+                    EncodeBson(DecodeExtendedJson(cEJ)).Should().Equal(cB, "native_to_bson( json_to_native(cEJ) ) = cB");
+                }
+            }
+
+            if (dB != null)
+            {
+                EncodeBson(DecodeBson(dB)).Should().Equal(cB, "native_to_bson( bson_to_native(dB) ) = cB");
+            }
+
+            if (dEJ != null)
+            {
+                EncodeCanonicalExtendedJson(DecodeExtendedJson(dEJ)).Should().Be(cEJ, "native_to_canonical_extended_json( json_to_native(dEJ) ) = cEJ");
+                if (!test.GetValue("lossy", false).ToBoolean())
+                {
+                    EncodeBson(DecodeExtendedJson(dEJ)).Should().Equal(cB, "native_to_bson( json_to_native(dEJ) ) = cB");
+                }
+            }
+
+            if (rEJ != null)
+            {
+                EncodeRelaxedExtendedJson(DecodeExtendedJson(rEJ)).Should().Be(rEJ, "native_to_relaxed_extended_json( json_to_native(rEJ) ) = rEJ");
+            }
+        }
+
+        private string UnescapeUnicodeCharacters(string value)
+        {
+            var pattern = @"\\u[0-9a-fA-F]{4}";
+            var unescaped = Regex.Replace(value, pattern, match =>
+            {
+                var bytes = BsonUtils.ParseHexString(match.Value.Substring(2, 4));
+                var c = (char)(bytes[0] << 8 | bytes[1]);
+                return c < 0x20 ? match.Value : new string(c, 1);
+            });
+            return unescaped;
+        }
+
+        // nested types
+        private class TestCaseFactory : JsonDrivenTestCaseFactory
+        {
+            #region static
+            private static readonly string[] __ignoredTestNames =
+            {
+                "dbpointer.json", // dbpointer not supported
```

Comment:
Small change request: add a `:` to the string to emphasize that ALL tests in the file are to be skipped, and not just an individual test.

    "dbpointer.json:", // dbpointer not supported

Reply:
Done.